src/unity/toolkits/supervised_learning/linear_svm_opt_interface.hpp

```diff
@@ -157,6 +164,9 @@ class linear_svm_scaled_logistic_opt_interface: public
   */
   size_t num_classes() const;
 
+  variant_type get_validation_accuracy();
```

Comment:
I think we should have these return a double instead of a variant type, since their name sorta conveys they return a numeric value.  You should only have to change this in this signature and the return code, though.

Reply:
That's a great point. Done.